tests/unit/test_energy_spectrum.py

```diff
@@ -271,5 +255,181 @@ def test_when_no_foil_is_true(self):
         )
 
 
+class TestSumSpectra(unittest.TestCase):
+
+    def create_sum_and_energy_spectrum(self, tmp_dir):
+        tmp_dir = Path(tmp_dir)
+        mesu = mo.get_measurement(
+            path=tmp_dir / "mesu.info", save_on_creation=True)
+        es = self.get_energy_spectra
+        sum_spectrum = SumEnergySpectrum(
+            es, mesu.directory, SumSpectrumType.MEASURED)
```

Comment:
In my opinion, the sum_spectrum should be created in the tests and not in this helper. This would give more control over the parameters passed to SumEnergySpectrum (es and SumSpectrumType) and make it more obvious what is being tested.